Makefile

```diff
@@ -2,9 +2,10 @@
 RM := rm -f
 SSH_USER := core
 TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
-DCOS_VERSION := 1.11
-KUBERNETES_VERSION ?= 1.10.7
-KUBERNETES_FRAMEWORK_VERSION ?= 1.2.2-1.10.7
+DCOS_VERSION := 1.12
+CUSTOM_DCOS_DOWNLOAD_PATH := https://downloads.dcos.io/dcos/testing/1.12.0-rc2/dcos_generate_config.sh
```

Comment:
This should use the stable relase ` https://downloads.dcos.io/dcos/stable/1.12.0/dcos_generate_config.sh`